private/buf/cmd/buf/buf_test.go

```diff
@@ -1027,6 +1028,518 @@ func TestCheckLsBreakingRulesFromConfigExceptDeprecated(t *testing.T) {
 	}
 }
 
+func TestLsModulesWorkspaceV1(t *testing.T) {
+	// Cannot be parallel since we chdir.
+	pwd, err := osext.Getwd()
+	require.NoError(t, err)
+	defer func() {
+		r := recover()
+		assert.NoError(t, osext.Chdir(pwd))
+		if r != nil {
+			panic(r)
+		}
+	}()
+
+	require.NoError(t, osext.Chdir(filepath.Join(pwd, "testdata", "lsmodules", "workspacev1")))
+	testRunStdout(
+		t,
+		nil,
+		0,
+		// default format is path
+		`
+a_v1
+b_no_name_v1
+c_v1beta1
+d_no_file
+e_no_file
+f_no_name_v1beta1
+`,
+		"config",
+		"ls-modules",
+	)
+	testRunStdout(
+		t,
+		nil,
+		0,
+		// sort by path
+		`
+a_v1
+b_no_name_v1
+c_v1beta1
+d_no_file
+e_no_file
+f_no_name_v1beta1
+`,
+		"config",
+		"ls-modules",
+		"--format",
+		"path",
+	)
+	testRunStdout(
+		t,
+		nil,
+		0,
+		// sort by name
+		`
+buf.build/bar/baz
+buf.build/foo/bar
+`,
+		"config",
+		"ls-modules",
+		"--format",
+		"name",
+	)
+	testRunStdout(
+		t,
+		nil,
+		0,
+		// sort by path
```

Comment:
```suggestion
		// format as json
```

Reply:
resolved in [baefb5d](https://github.com/bufbuild/buf/pull/3081/commits/baefb5d1126e8d4e89b1100036331a9f2ce83006)